test/tests/functional/pbs_ralter.py

```diff
@@ -476,7 +476,7 @@ def test_alter_advance_resv_start_time_before_run(self):
 
         All the above operations are expected to be successful.
         """
-        offset = 20
+        offset = 40
```

Comment:
I'd bump this to 60.  I modified a similar test last week and used 60.  Since this is the second test that needs modification, you might consider looking at other similar tests.  Look for tests that don't care about when the reservation starts but wants to alter it before it starts.